src/pages/Events/EventDoorCounter.tsx

```
@@ -0,0 +1,210 @@
+        </div>
+
+        {/* Counter Dashboard */}
+        <div className="bg-slate-900/80 border border-blue-500/20 rounded-3xl p-8">
+          <div className="flex flex-col md:flex-row items-center justify-between gap-6">
+            <div className="text-center md:text-left">
+              <h2 className="text-2xl font-bold flex items-center gap-2"><DoorOpen className="w-6 h-6 text-blue-400" /> Door Counter</h2>
+              <p className="text-slate-400 text-sm mt-2">Manually adjust the count or let the real-time simulation update it.</p>
+            </div>
+            
+            <div className="flex items-center gap-4">
+              <button 
```

ESLint

[error] Delete `·`

(prettier/prettier)